.gitignore

```diff
@@ -29,6 +29,7 @@ pip-log.txt
 
 # Mac OSX
 .DS_Store
+.icloud
```

Comment:
This was annyoing on my side :)

Reply:
I'm not using it, but clearly good enough reason to ignore it!